src/RestControllers/InsuranceRestController.php

```diff
@@ -32,7 +32,7 @@ public function getAll($pid)
 
     public function getOne($pid, $type)
     {
-        $serviceResult = $this->insuranceService->getOne($pid, $type);
+        $serviceResult = $this->insuranceService->getOneByPid($pid, $type);
```

Comment:
the rest controllers should not be changed (the rest endpoints will use uuid then)
Just need to ensure all the other placed in codebase that use the service are changed

Reply:
actually, this is more complicated since the endpoint takes in pid and type. let me look into that to see best strategy.

Reply:
TODO for me: see above

Reply:
ok, lets also keep this as is (getOneByPid). And just need to change one more instance of getOne (to getOneByPid) use in the interface/billing/sl_eob_process.php

Reply:
Changed the functionname in interface/billing/sl_eob_process.php